mypy/stubgen.py

```diff
@@ -399,9 +547,50 @@ def process_namedtuple(self, lvalue: NameExpr, rvalue: CallExpr) -> None:
         else:
             items = '<ERROR>'
         self.add('%s = namedtuple(%s, %s)\n' % (lvalue.name, name, items))
-        self._classes.add(lvalue.name)
         self._state = CLASS
 
+    def is_type_expression(self, expr: Expression, top_level: bool=True) -> bool:
```

Comment:
I can add a test for the former...
The latter will actually go through as an alias and I'm not sure how to prevent it with the kind of information I have in stubgen (I'm assuming that in most libraries that kind of things won't happen too much at a module top level).